src/main/java/org/example/service/NotFoundService.java

```diff
@@ -0,0 +1,33 @@
+package org.example.service;
+
+public class NotFoundService {
+
+    private static NotFoundService INSTANCE;
```

Comment:
Check the naming conventions.
When the variable title should be written in caps.

Reply:
fixed